packages/manager/src/constants/API_ENDPOINTS.ts

```diff
@@ -41,14 +41,29 @@ export const API_ENDPOINTS: APIEndpoints = (() => {
 
 			if (missingAPIEndpoints.length > 0) {
 				console.error(
-					`You are running Slice Machine in development mode where API endpoints are configured via environment variables. The following endpoints were not configured: ${missingAPIEndpoints.join(
-						", ",
-					)}. Configure them before continuing.`,
+					`You are running Slice Machine in development mode (SM_ENV=${
+						APPLICATION_MODE.Development
+					}) where API endpoints are configured via environment variables.
+
+The following endpoints were not configured: ${missingAPIEndpoints.join(", ")}.
+
+Configure them before continuing.`,
 				);
 
 				process.exit(1);
 			}
 
+			console.warn(`You are running Slice Machine in development mode (SM_ENV=${
+				APPLICATION_MODE.Development
+			}).
+
+The following API endpoints were configured via environment variables:
+${Object.entries(apiEndpoints)
+	.map(([name, endpoint]) => `  - ${name}: ${endpoint}`)
+	.join("\n")}
+
```

Comment:
To be extra clear to users, maybe we could add a line like this if we detect the endpoints are different than prod or staging:

> These endpoints are different than Slice Machine's normal endpoints and are not trusted.

Reply:
I added it after the list, thanks!

I don't think we want to run some logic to differentiate weird endpoints from others (feeling like it would be a bit overkill) as users should not be using this mode anyway.